src/App.tsx

```diff
@@ -1,61 +1,142 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { ChangeEvent, useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todos } from './components/TodoInfo';
+
+function getUserById(userId: number) {
+  return usersFromServer.find((user) => user.id === userId);
```

Comment:
```suggestion
  return usersFromServer.find((user) => user.id === userId) || null;
```
there can be no user